src/components/Tabs/Tabs.tsx

```diff
@@ -237,24 +197,23 @@ export const Tabs = ({
         ref={headerRef}
       >
         <ul className={styles['tabs__list']} role="tablist">
-          {getTabs().map((tab, i) => {
+          {tabs.map((tab, i) => {
             const isActive = activeIndexState === i;
             return (
               <li
                 className={clsx(
                   styles['tabs__item'],
                   isActive && styles['eds-is-active'],
                 )}
-                key={'tabs-item-' + i}
+                key={tabIds[i]}
                 role="presentation"
               >
                 <a
-                  aria-controls={tabPanelIds[i]}
+                  aria-controls={activeTabPanelId}
                   aria-selected={isActive}
                   className={styles['tabs__link']}
-                  href={`#${tabPanelIds[i]}`}
+                  href={`#${activeTabPanelId}`}
```

Comment:
All tabs now have their `href` pointing to the lone active tab panel id.

Maybe that's weird, maybe it's not. Either way, a future PR is going to change these to button elements, since they're not actually links.